.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Test
       run: dotnet test --configuration Release --no-build
     - name: Pack
-      run: dotnet pack --configuration Release --no-build --output packages
+      run: dotnet pack --configuration Release --no-build --output packages /p:Version=1.0.0
```

Comment:
This should probably be in a root Directory.Build.props file or other .props file